### DISCUSSION TITLE
pageviews

### DISCUSSION THREAD

**Discussion by rimzzlabs** [umami-software/umami#1060] [Q&A] [answered]
Hello, I have a problem working with the API request, I want to get the pageviews from the umami app.
the URL was using a subdomain like `https://umami.mydomain.`, and the app was deployed to Vercel, I've successfully get the token by making a POST request with the credential _username and password_.

But when I try to making a GET request to `api/website/1/pageviews` with the token on the header, all I got was only _**400 bad request**_.
I was getting CORS error too (making a request on localhost btw)
please help 🙏

**Comment by mikecao** [comment, marked as answer]
I think I need to enable CORS on more endpoints. So far it's only on `/collect`. Will be available in the next release.
